backend/Recodo.API/Controllers/AccessToVideoController.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Recodo.BLL.Services;
+using Recodo.Common.Dtos.Access;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Recodo.API.Controllers
+{
+    [Route("api/access")]
+    [ApiController]
+    public class AccessToVideoController : ControllerBase
+    {
+        private readonly AccessToVideoService _accessToVideoService;
+        public AccessToVideoController(AccessToVideoService accessToVideoService)
+        {
+            _accessToVideoService = accessToVideoService;
+        }
+
+        [HttpPut]
+        public async Task<IActionResult> AddNewAccess(AccessForUnregisteredUsersDTO accessForUnregisteredUsersDTO)
+        {   
+            await _accessToVideoService.AddUserAccess(accessForUnregisteredUsersDTO.Email, accessForUnregisteredUsersDTO.VideoId);
+            return Ok();
+        }
+
+        [HttpPost]
```

Comment:
```suggestion
        [HttpGet]
```

Reply:
Yes, i wanted to add there HttpGet, i will add new method to our resourse service to get something not only by id, and then change in to HttpGet.